README.md

```diff
@@ -1,24 +1,198 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+<h1> <b> Rails - Blog-App</b> <hr/>
+   <img src="ruby.png" alt="Ruby Language Icon" height="400px" />
```

Comment:
- [ ] [OPTIONAL]
Kindly endeavor to fix the broken logo image link. This will make the ReadMe appear more professional!

**Broken lego link:**
![image](https://github.com/ZewdieMc/blog-app/assets/12745474/14a01dc8-909f-4164-b53b-71c1e244b3e0)